src/utils.ts

```diff
@@ -325,6 +326,7 @@ export const findIndexInSortedArray = <T, L>({
    * selectValueToCompare: (message) => message.created_at.getTime()
    * ```
    */
+  selectKey?: (arrayElement: T) => string;
```

Comment:
Could you please add a description for this property?

Reply:
Added in https://github.com/GetStream/stream-chat-js/pull/1421/commits/41275d9bef77bcac6caec589031d11932dc2c6d6.